website/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@docusaurus/core": "^2.4.1",
     "@docusaurus/plugin-content-docs": "^2.4.1",
+    "@docusaurus/plugin-google-tag-manager": "^2.4.1",
```

Comment:
Seems unnecessary to add this plugin considering it's included in the [classic preset](https://docusaurus.io/docs/using-plugins#docusauruspreset-classic), no?

Reply:
Just following directions from the docusaurus page 🤷🏽‍♀️. Unfortunately, no way to test the traffic locally.